wget/plan.sh

```diff
@@ -1,22 +1,18 @@
 pkg_name=wget
-pkg_distname=$pkg_name
+pkg_distname="${pkg_name}"
 pkg_origin=core
 pkg_version=1.18
 pkg_license=('gplv3+')
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
-pkg_source=http://ftp.gnu.org/gnu/$pkg_distname/${pkg_distname}-${pkg_version}.tar.xz
+pkg_source=http://ftp.gnu.org/gnu/${pkg_distname}/${pkg_distname}-${pkg_version}.tar.xz
 pkg_shasum=b5b55b75726c04c06fe253daec9329a6f1a3c0c1878e3ea76ebfebc139ea9cc1
 pkg_deps=(core/glibc core/libidn core/zlib core/openssl core/cacerts)
-pkg_build_deps=(core/coreutils core/diffutils core/patch core/make core/gcc core/sed core/grep core/pkg-config)
+pkg_build_deps=(core/diffutils core/patch core/make core/gcc core/sed core/grep core/pkg-config)
 pkg_bin_dirs=(bin)
 
-do_prepare() {
```

Comment:
We still want this function (and call `_wget_common_prepare`) as [core/wget-static](https://github.com/habitat-sh/core-plans/blob/2016e02cda556eaad3b56bb40477144757ed8ddf/wget-static/plan.sh#L22) uses it. IIRC without preparing the source tree in this way, wget was not buildable in a Studio.